### PR TITLE
Migrate Network Config Modal from chakra to shadcn/tailwind

### PR DESCRIPTION
**Pull request by MananTank**
## Problem solved

Short description of the bug fixed or feature added

---

## PR-Codex overview
This PR removes deprecated components, updates UI elements, and refactors form fields in the dashboard. 

### Detailed summary
- Removed deprecated components: `TooltipBox.tsx`, `ConfirmationPopover.tsx`, `thirdweb.client.ts`
- Updated UI elements in various components
- Refactored form fields in `ChainIdInput.tsx`, `RpcInput.tsx`, `NetworkIdInput.tsx`, and `FormFieldSetup.tsx`

> The following files were skipped due to too many changes: `apps/dashboard/src/@/components/ui/radio-group.tsx`, `apps/dashboard/src/components/configure-networks/ConfigureNetworkForm.tsx`

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

**Comment by MananTank** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/thirdweb-dev/js/4745?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#4745** <a href="https://app.graphite.dev/github/pr/thirdweb-dev/js/4745?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#4743** <a href="https://app.graphite.dev/github/pr/thirdweb-dev/js/4743?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @MananTank and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: apps/dashboard/src/@/components/blocks/FormFieldSetup.tsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import { Label } from "@/components/ui/label";
+import { ToolTipLabel } from "@/components/ui/tooltip";
+import { AsteriskIcon, InfoIcon } from "lucide-react";
+
+export function FormFieldSetup(props: {
+  id: string;
```
Comment (jnsdls) [inline apps/dashboard/src/@/components/blocks/FormFieldSetup.tsx:6]: probably copy the `htmlFor` name here to make it clear what it is used for?

File: apps/dashboard/src/components/configure-networks/ConfigureNetworkForm.tsx [hunk 2]
```diff
@@ -1,27 +1,22 @@
+import { FormFieldSetup } from "@/components/blocks/FormFieldSetup";
+import {} from "@/components/ui/alert";
```
Comment (jnsdls) [inline apps/dashboard/src/components/configure-networks/ConfigureNetworkForm.tsx:2]: remove

**Review by jnsdls — CHANGES_REQUESTED** [review]

File: apps/dashboard/src/@/components/blocks/FormFieldSetup.tsx (hunk 1 above)
Comment (MananTank) [reply apps/dashboard/src/@/components/blocks/FormFieldSetup.tsx]: updated

File: apps/dashboard/src/components/configure-networks/ConfigureNetworkForm.tsx (hunk 2 above)
Comment (MananTank) [reply apps/dashboard/src/components/configure-networks/ConfigureNetworkForm.tsx:2]: removed

**Review by jnsdls — APPROVED** [review]

File: apps/dashboard/src/components/configure-networks/ConfigureNetworkForm.tsx (hunk 2 above)
Comment (MananTank) [reply apps/dashboard/src/components/configure-networks/ConfigureNetworkForm.tsx]: removed for real now

**Pull request merged**